cmd/destroy.go

```diff
@@ -131,25 +132,28 @@ if the registry has already been deleted.`,
 
 		log.Println("gitlab terraform destruction complete")
 
-		log.Println("disabling ArgoCD auto sync")
-		argoCDUsername := viper.GetString("argocd.admin.username")
-		argoCDPassword := viper.GetString("argocd.admin.password")
+		//This should wrapped into a function, maybe to move to: k8s.DeleteRegistryApplication
```

Comment:
side note; it would be nice to stick in the cmd/driver, move functions into smaller functions is nice, but from an architecture perspective, when we introduce methods, it will become "macarrônico".

Reply:
I am not getting why functions are bad... this should be part of delete register, as this  happens only `k8s.DeleteRegistryApplication` is executed. 

I am don't share what arch choice you are referring, anyway.. original code is breaking destroy. We are proceeding with merge to unblock some work. ;)

Reply:
I just want want stuff that run together, inside the same "if" to be near, so the behavior is consistent.